src/common/providers/execution/execution.ts

```diff
@@ -140,10 +140,10 @@ export class Execution {
       gasLimit: this.config.get('TX_GAS_LIMIT'),
     });
     context = { ...context, tx: populated };
+    const isFeePerGasAcceptable = await this.isFeePerGasAcceptable();
```

Comment:
Are you trying to log the value? If so why?

Reply:
To have it closer to the normal execution: it should call the

Reply:
To have dry run as much close to the normal execution as possible: it do have to call gas fee history  to be sure that it will work in production on some specific node, for example. We also will have prod-like logs on our staging instance